packages/react/src/definitions/user.ts

```diff
@@ -7,7 +7,7 @@ import { TransactionFilterKey } from './transaction';
 export const UserUrl = {
   get: 'user',
   ref: 'user/ref',
-  delete: 'user',
+  delete: 'user/account',
```

Comment:
Is this right?

Reply:
Should be correct I think 🤔

https://github.com/DFXswiss/api/blob/f25394de0ad7680e029c0b2dcd9697eb02520196/src/subdomains/generic/user/models/user/user.controller.ts#L254

Reply:
Ah yes, just saw it. But wouldn't it make sense to have it on `DELETE user` in the backend? Would be a better fit to the rest of the paths.

Reply:
I'm good with that too 👍 @Yannick1712?

Reply:
Yes that's right we changed that 👍 

@davidleomay Hm your deactivating the userAccount (userData) with that user endpoint. For me it's a bit strange to have a `Delete()` on user controller, which "deletes"/deactivates not the user but the userData instead 🤔 that's why for me `Delete(account)` makes more sense in the user controller. 
But if both of you think that `Delete()` would be better, I'm fine with that ✌️

Reply:
@lapatric let's change it to `user` only 😉